Kconfig.zephyr

```diff
@@ -883,6 +883,16 @@ config MCUBOOT_GENERATE_CONFIRMED_IMAGE
 	  The existence of bin and hex files depends on CONFIG_BUILD_OUTPUT_BIN
 	  and CONFIG_BUILD_OUTPUT_HEX.
 
+config MCUBOOT_GENERATED_HEX_BIN_SAME_CONTENT
+	bool "Ensure that generated .hex and .bin files have the same logical content"
+	default y
```

Comment:
Quite honestly I don't think the majority of people care about this, some yes but the minority.